diff --git a/node_modules/simple-hypercore-protocol/lib/handshake.js b/node_modules/simple-hypercore-protocol/lib/handshake.js
index efc583d..c7a35e8 100644
--- a/node_modules/simple-hypercore-protocol/lib/handshake.js
+++ b/node_modules/simple-hypercore-protocol/lib/handshake.js
@@ -1,5 +1,5 @@
 const SH = require('simple-handshake')
-const DH = require('noise-protocol/dh')
+const DH = require('noise-protocol/dh')()
 const crypto = require('hypercore-crypto')
 const varint = require('varint')
 
